src/yunohost/firewall.py

```diff
@@ -327,66 +331,57 @@ def firewall_upnp(action="status", no_refresh=False):
 
     """
     firewall = firewall_list(raw=True)
-    enabled = firewall["uPnP"]["enabled"]
-
-    # Compatibility with previous version
-    if action == "reload":
-        logger.debug("'reload' action is deprecated and will be removed")
-        try:
-            # Remove old cron job
-            os.remove("/etc/cron.d/yunohost-firewall")
-        except Exception:
-            pass
-        action = "status"
-        no_refresh = False
 
-    if action == "status" and no_refresh:
-        # Only return current state
-        return {"enabled": enabled}
-    elif action == "enable" or (enabled and action == "status"):
+    if action == "status":
+        enabled = firewall["uPnP"]["enabled"]
+    elif action == "enable":
         # Add cron job
         with open(UPNP_CRON_JOB, "w+") as f:
             f.write(
-                "*/50 * * * * root "
+                "*/10 * * * * root "
```

Comment:
Changing to a frequency of once per 12 minutes would reduce the possibility of a clash - is this ok?

There is an alternative to repeatedly opening the port here:

https://github.com/mqus/nft-rules/blob/master/files/SSDP_client.md

This would involve using nft rules - is this acceptable?

I agree that the command name is misleading, but thought it risky to change it if it works...